packages/apollo-server-core/src/caching.ts

```diff
@@ -0,0 +1,12 @@
+export interface KeyValueCache {
+  set(
+    key: string,
+    data: string,
+    options?: {
+      ttl?: number;
+      tags?: string[];
+    },
+  ): Promise<void>;
+  get(key: string): Promise<string>;
+  invalidateTags(tags: string[]): Promise<void>;
```

Comment:
What is this? It doesn't seem useful for PQ caches (which are hash-based), so it just seems like it might create a larger requirement for implementing an appropriate cache. Same with TTL.